pkg/version/version.go

```diff
@@ -31,7 +31,7 @@ var (
 	State       string
 
 	// MaxPluginVersion must be increased to add new plugin and make sure the new plugin version is same to this value
-	MaxPluginVersion = 2
+	MaxPluginVersion = 3
```

Comment:
I am not sure that I understand. Why can't dataversion 2 can't support rename? Did you change anything in such a way that it is not possible?